lib/perl/Genome/Role/Comparable/Command/DiffBlessed.pm

```diff
@@ -11,6 +11,14 @@ role Genome::Role::Comparable::Command::DiffBlessed {
     requires => ['new_object'],
 };
 
+sub help_brief : Overrides(Genome::Role::Comparable::Command::Diff) {
```

Comment:
I'm not sure what it means for this to specify an override for a role that it does not itself consume.